app/assets/javascripts/angualr/controllers/people.js.erb

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function(){
+  
+  // Scroll to the Previous Board Group Picture
+  $(document).on('click', '.2013-2014', function(){   
+    $("html, body").animate({             
+      scrollTop: $("#2013-2014").offset().top - 150
+    }, 1000);     
+  });
+
+  $(document).on('click', '.2012-2013', function(){   
+    $("html, body").animate({             
+      scrollTop: $("#2012-2013").offset().top - 150
+    }, 1000);     
+  });
+
+  $(document).on('click', '.2011-2012', function(){   
+    $("html, body").animate({             
+      scrollTop: $("#2011-2012").offset().top - 150
+    }, 1000);     
+  });
+
+  $(document).on('click', '.2010-2011', function(){   
+    $("html, body").animate({             
+      scrollTop: $("#2010-2011").offset().top - 150
+    }, 1000);     
+  });
+})
+
+app.controller('peopleCtrl', ['$scope', function($scope){
```

Comment:
very skinny controllers in angular no? hehehe come on bill, add a bit of functionality in angular too k?